stage/services/bastion/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_instance" "bastion-stage" {
+    ami = "${var.ami["eu-west-1"]}"
+    instance_type = "t2.micro"
+    vpc_security_group_ids = ["${aws_security_group.bastion.id}"]
+    subnet_id = "${data.aws_subnet.datacite-public.id}"
+    key_name = "${var.key_name}"
+    associate_public_ip_address = "true"
+    user_data = "${data.template_file.bastion-user-data-cfg.rendered}"
+    tags {
+        Name = "Bastion-Stage"
+    }
+}
+
+resource "aws_eip" "bastion-stage" {
```

Comment:
Please check whether we have free elastic IPs available. There is a hard limit per AWS region (normally 5, we are currently at 8).

Reply:
We have 6 so I guess I could allocate another? or is there an alternative.

Reply:
That is fine. An elastic IP might not be so important here, but keeps this in sync with the production configuration.